pkg/controllers/vdb/restart_reconciler.go

```diff
@@ -126,9 +126,14 @@ func (r *RestartReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctr
 	// admin commands to run.  Cluster operations only apply if the entire
 	// vertica cluster is managed by k8s.  We skip that if initPolicy is
 	// ScheduleOnly.
-	if r.PFacts.getUpNodeAndNotReadOnlyCount() == 0 &&
+	if r.PFacts.GetUpNodeAndNotReadOnlyCount() == 0 &&
 		r.Vdb.Spec.InitPolicy != vapi.CommunalInitPolicyScheduleOnly {
-		return r.reconcileCluster(ctx)
+		// Abort cluster restart if at least one pod must not be
+		// restarted.
+		if r.PFacts.GetShutdownCount() == 0 {
+			return r.reconcileCluster(ctx)
+		}
+		return ctrl.Result{}, nil
```

Comment:
What if GetShutdownCount() < clusterSize? Do we need to support stop part of the sandbox?

Reply:
This path is for `startdb`. We don't allow `startdb` if there is at least one subcluster with `shutdown` set to true. Keep in mind we will support `stop_subcluster` so it will be possible to stop only specific subcluster(s), although realdb does not have that use case.